### ISSUE TITLE
TextField titleLabel not working when using Storyboards

### ISSUE THREAD

**Issue by sverin** [CosmicMind/Material#241]
I'm glad that you exposed the properties with @IBInspectable.

However, when I use the TextField in storyboard and set the properties in the designer, I can't get the title label to work. I have to link the textfield from the storyboard and add the UILabel to the textfield manually.
Is this intentional?

I got it working by doing this and configure the rest in IB:

```
@IBOutlet weak var myTextField: MyTextField!

override func viewDidLoad() {
    myTextField.titleLabel = UILabel()
}
```

**Comment by danieldahan** [comment]
I am going to make an update that will avoid the setup. There are a few TextField tasks that are lined up and I will do them all at once. I will leave this up for now. Thank you for submitting this.

**Comment by sverin** [comment]
Thank, looking forward to it

**Comment by danieldahan** [comment]
This is now fixed in 1.39.0. You no longer need to setup the titleLabel. :)
